frontend/src/pages/AccountSent.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <q-page padding>
+    <h2 class="page-title">{{ $t('AccountSendTable.sent-funds') }}</h2>
+    <!-- User has not connected wallet  -->
+    <div v-if="!userAddress">
+      <p class="text-center">{{ $t('AccountSent.connect-your-wallet') }}</p>
+      <div class="row justify-center">
+        <connect-wallet>
+          <base-button class="text-center" :label="$t('AccountSent.connect-wallet')" />
+        </connect-wallet>
+      </div>
+    </div>
+    <div v-else class="q-mx-auto" style="max-width: 800px">
+      <account-send-table :sendMetadata="sendMetadata" />
+    </div>
+  </q-page>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref } from 'vue';
+import useWallet from 'src/store/wallet';
+import useWalletStore from 'src/store/wallet';
+import AccountSendTable from 'components/AccountSendTable.vue';
+import ConnectWallet from 'components/ConnectWallet.vue';
+import { SendTableMetadataRow } from 'components/models';
+import { BigNumber } from 'src/utils/ethers';
+import { formatNameOrAddress } from 'src/utils/address';
+import { formatDate, formatAmount, formatTime, getTokenSymbol, getTokenLogoUri } from 'src/utils/utils';
+import { NATIVE_TOKEN_ADDRESS } from 'components/models';
+
+function useAccountSent() {
+  const { userAddress } = useWallet();
+  const { tokens } = useWalletStore();
+  const sendMetadata = ref<SendTableMetadataRow[]>([]);
+
+  try {
+    const formattedRows = [];
+    const data = [
+      {
+        amount: '1000000000000000000',
+        tokenAddress: NATIVE_TOKEN_ADDRESS,
+        dateSent: new Date(),
+        address: '0x2436012a54c81f2F03e6E3D83090f3F5967bF1B5',
+        hash: '0xabeea5720640859db6dc46caae44fe34449469bfae8e3e930d9f59abbf50ed50',
+      },
+    ];
+
+    for (const row of data) {
+      formattedRows.push({
```

Comment:
Wdyt about generating the `formattedRows` array with `data.map` and using that to initialize `sendMetadata`, and removing the try/catch since I don't think we really need that here? Would make this a bit more concise and remove the need to initialize an empty array